packages/plugins/contracts/test/script/Simple.s.sol

```diff
@@ -12,8 +12,7 @@ contract Simple1 is Script, Sphinx {
         sphinxConfig.owners = [0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266];
         sphinxConfig.threshold = 1;
 
-        sphinxConfig.mainnets = [Network.ethereum, Network.optimism];
-        sphinxConfig.testnets = [Network.sepolia];
+        sphinxConfig.mainnets = ["ethereum", "optimism_mainnet"];
```

Comment:
Changed "optimism" to "optimism_mainnet" to test that we can propose/deploy against non-standard network names. This change is reflected in the foundry.toml in the plugins package